frontend/icons/src/main/java/icons/MacheteIcons.java

```diff
@@ -13,35 +13,35 @@ private MacheteIcons() {}
 
   public static final Icon DISCOVER = loadIcon("/macheteLogoIcon.svg");
 
+  public static final Icon EDIT = loadIcon("/edit.svg");
+
   public static final Icon FAST_FORWARD = loadIcon("/applyNotConflictsLeft.svg");
 
-  public static final Icon FETCH = loadIcon("/download.svg");
+  public static final Icon FETCH = loadIcon("/fetch.svg");
 
   public static final Icon HELP = loadIcon("/help.svg");
 
   public static final Icon MACHETE_FILE = loadIcon("/macheteLogoIcon.svg");
 
-  public static final Icon MACHETE_LOAD_CHANGES = loadIcon("/macheteLoadChanges.svg");
+  public static final Icon MERGE_PARENT = loadIcon("/merge.svg");
 
   public static final Icon OVERRIDE_FORK_POINT = loadIcon("/overrideForkPoint.svg");
 
-  public static final Icon PULL = loadIcon("/incomingChangesOn.svg");
+  public static final Icon PULL = loadIcon("/pull.svg");
 
-  public static final Icon PUSH = loadIcon("/outgoingChangesOn.svg");
+  public static final Icon PUSH = loadIcon("/push.svg");
 
-  public static final Icon REBASE = loadIcon("/menu-cut.svg");
+  public static final Icon REBASE = loadIcon("/menuCut.svg");
```

Comment:
Nit, for consistency... the other icons AFAICS are called after the git-machete purpose they serve, not the original IntelliJ purpose
```suggestion
  public static final Icon REBASE = loadIcon("/rebase.svg");
```